src/main/java/org/broadinstitute/hellbender/tools/exome/orientationbiasvariantfilter/OrientationBiasUtils.java

```diff
@@ -248,17 +249,27 @@ static long calculateNumNotTransition(final String sampleName, final List<Varian
                 .filter(g -> OrientationBiasUtils.isGenotypeInTransition(g, complement) || OrientationBiasUtils.isGenotypeInTransition(g, transition));
     }
 
+    private static Stream<Genotype> getNumArtifactGenotypeStreamKeepingOBFilteredVCs(String sampleName, List<VariantContext> variantContexts, Transition transition, Transition complement) {
```

Comment:
Does `complement` really need to be an argument i.e. how much do you save by not just calculating on the fly?